### DISCUSSION TITLE
TailwindCSS and PostCSS Nest not working

### DISCUSSION THREAD

**Discussion by ric980** [tailwindlabs/tailwindcss#10630] [Help] [answered]
Hi there,

I cannot get PostCSS Nested to work with Tailwind CSS.

This is my postcss config:
```
module.exports = {
  plugins: {
    'postcss-import': {},
    'tailwindcss/nesting': {},
    tailwindcss: {},
    autoprefixer: {}
  }
}
```

This is my CSS:
```
.prose {
  @apply mx-auto max-w-3xl;

  h1 {
    @apply mb-12;
  }

  h2 {
    @apply mt-12 mb-3;
  }

  h3 {
    @apply mt-6 mb-3;
  }

  p {
    @apply my-3;
  }
}
```

And this is what comes out in the browser:
<img width="235" alt="Screenshot 2023-02-19 at 17 55 56" src="https://user-images.githubusercontent.com/121663568/219959323-19c9cb19-70a2-4e9e-8449-0eb3a8dedd97.png">

It seems like Tailwind and postcss-nested doesn't work together. It's a Remix project if that matters.

**Comment by adamwathan** [comment, marked as answer]
Hey! Please provide a reproduction of the problem, there are so many variables in how you could have things set up here that it’s not really possible to help without being able to clone a project that reproduces the issue.

My guesses:

1. Maybe you are using the Tailwind CLI tool and forgetting to use the `--postcss` flag when invoking it to make sure it reads your PostCSS config file.
2. Maybe you are relying on the new unstable automatic Tailwind CSS support in Remix but haven’t added the necessary flag for PostCSS support to your Remix config (haven’t actually tried this so not sure if this is needed or not.)

**Comment by ric980** [comment]
I didn't know at all about the `--postcss` flag. That was it. Thank you very much.
